### PR TITLE
Update preprocess bindings docstrings

### PR DESCRIPTION
**Pull request by pranjalnandi**

### CODE REVIEW

File: bindings/docstrings/nvdspreprocessmetadoc.h [hunk 1]
```diff
@@ -7,29 +7,43 @@ namespace pydsdoc
...
-                :ivar buffer_size: *int*, buffer_size
-                :ivar tensor_shape: *vector*, tensor_shape
-                :ivar data_type: *NvDsDataType*, data_type
-                :ivar tensor_name: *string*, tensor_name
-                :ivar gpu_id: *int*, gpu_id
-                :ivar private_data: *void*, private_data
-                :ivar meta_id: *int*, meta_id)pyds";
+                :ivar raw_tensor_buffer: Holds raw tensor buffer preprocessed for infer.
+                :ivar buffer_size: *int*, Size of raw tensor buffer.
+                :ivar tensor_shape: *vector*, Shape of raw tensor buffer.
```
Comment (rayed-therap) [inline bindings/docstrings/nvdspreprocessmetadoc.h]: vector?

File: bindings/docstrings/nvdspreprocessmetadoc.h [hunk 2]
```diff
@@ -7,29 +7,43 @@ namespace pydsdoc
...
+                :ivar data_type: *NvDsDataType*, Model datatype for which tensor prepared.
```
Comment (rayed-therap) [inline bindings/docstrings/nvdspreprocessmetadoc.h]: `:class:`

File: bindings/docstrings/nvdspreprocessmetadoc.h [hunk 3]
```diff
@@ -7,29 +7,43 @@ namespace pydsdoc
...
+                :ivar tensor_name: *string*, Model input layer name.
+                :ivar gpu_id: *int*, gpu-id on which tensor prepared.
+                :ivar private_data: Private data used for the meta producer's internal memory management.)pyds";
```
Comment (rayed-therap) [inline bindings/docstrings/nvdspreprocessmetadoc.h]: what is the type of private_data?

File: bindings/docstrings/nvdspreprocessmetadoc.h [hunk 4]
```diff
@@ -7,29 +7,43 @@ namespace pydsdoc
...
-                Holds information about preprocess batch metadata
+                Holds information about preprocess batch metadata in the preprocessed meta
                 
-                :ivar target_unique_ids: *int*, target_unique_ids
-                :ivar tensor_meta: *NvDsPreProcessTensorMeta*, tensor_meta
-                :ivar roi_vector: *vector*, roi_vector
-                :ivar private_data: *void*, private_data)pyds";
+                :ivar target_unique_ids: *int*, Holds the list of component gie-id for which tensor is prepared.
+                :ivar tensor_meta: Holds a pointer to object of type :class: NvDsPreProcessTensorMeta.
+                :ivar roi_vector: *vector*, Holds the list of RoiMeta per batch.
```
Comment (rayed-therap) [inline bindings/docstrings/nvdspreprocessmetadoc.h]: vector?

Comment (rayed-therap) [reply bindings/docstrings/nvdspreprocessmetadoc.h]: See how they write vector descriptions in python.

File: bindings/docstrings/nvdsroimetadoc.h [hunk 5]
```diff
@@ -6,39 +6,46 @@ namespace pydsdoc
...
+            constexpr const char* NvDsUnitType_FullFrame=R"pyds(metadata type to be set for Full Frames)pyds";
+            constexpr const char* NvDsUnitType_ROI=R"pyds(metadata type to be set for Region of Interests)pyds";
+            constexpr const char* NvDsUnitType_Object=R"pyds(metadata type to be set for Cropped Objects)pyds";
         }
         
         namespace NvDsRoiMetaDoc
         {          
             constexpr const char* descr = R"pyds(
-                Holds information about roi metadata
+                Holds information about roi metadata in the roi_vector of preprocessed batch meta.
```
Comment (rayed-therap) [inline bindings/docstrings/nvdsroimetadoc.h]: - undo this change
- add period

File: bindings/docstrings/nvdspreprocessmetadoc.h [hunk 6]
```diff
@@ -7,29 +7,43 @@ namespace pydsdoc
...
+                :ivar private_data: Private data used for the meta producer's internal memory management.)pyds";
             
             constexpr const char* cast=R"pyds(cast given object/data to :class:`NvDsPreProcessTensorMeta`, call pyds.NvDsPreProcessTensorMeta.cast(data))pyds";
         }
 
         namespace GstNvDsPreProcessBatchMetaDoc
         {
             constexpr const char* descr = R"pyds(
-                Holds information about preprocess batch metadata
+                Holds information about preprocess batch metadata in the preprocessed meta
```
Comment (rayed-therap) [inline bindings/docstrings/nvdspreprocessmetadoc.h]: - undo
- add period

File: bindings/docstrings/nvdspreprocessmetadoc.h [hunk 7]
```diff
@@ -7,29 +7,43 @@ namespace pydsdoc
         namespace NvDsPreProcessTensorMetaDoc
         {
             constexpr const char* descr = R"pyds(
-                Holds information about preprocessed tensor metadata
+                Holds information about preprocess tensor metadata in the preprocessed batch meta
```
Comment (rayed-therap) [inline bindings/docstrings/nvdspreprocessmetadoc.h]: the second part is redundant.
remove and add period.

**Review by rayed-therap — CHANGES_REQUESTED** [review]

File: bindings/docstrings/nvdspreprocessmetadoc.h (hunk 3 above)
Comment (pranjalnandi) [reply bindings/docstrings/nvdspreprocessmetadoc.h]: nvidia skipped to mention type if it is void. private data is void type.
